cmd/endorse.go

```diff
@@ -180,16 +198,37 @@ func (f *endorseCommand) InitContext(ctx context.Context) (context.Context, erro
 	if err != nil {
 		return nil, fmt.Errorf("could not read UEFI file %s: %v", f.UefiPath, err)
 	}
+	if f.SvsmPath != "" {
+		ec.SvsmImage, err = ops.ReadFile(ctx, f.Storage, "", f.SvsmPath)
```

Comment:
Please define a constant like noBucket = "" to use instead of a mysterious empty string literal argument.